h2o-core/src/main/java/water/Job.java

```diff
@@ -52,7 +52,7 @@ public Job(Key<T> key, String clz_of_T, String desc) {
     super(defaultJobKey());     // Passing in a brand new Job key
     assert key==null || clz_of_T!=null;
     _result = key;              // Result (destination?) key
-    _typeid = clz_of_T==null ? 0 : TypeMap.onIce(clz_of_T);
+    _typeid = clz_of_T==null ? 0 : TypeMap.getIcedId(clz_of_T);
```

Comment:
@mmalohlava what do you think about this change?

Can we assume the class is already in the TypeMap when the constructor is called?